### PR TITLE
Create a (dummy) public page for tools

### PR DESCRIPTION
**Pull request by pysailor**
Still needs to be designed
Refs #SCR-958

### CODE REVIEW

File: src/euphorie/client/browser/templates/oira_tool.pt [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      xmlns:meta="http://xml.zope.org/namespaces/meta"
```
Comment (ale-rt) [inline src/euphorie/client/browser/templates/oira_tool.pt:4]: Not needed

File: src/euphorie/client/browser/templates/oira_tool.pt [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      id="sessions"
```
Comment (ale-rt) [inline src/euphorie/client/browser/templates/oira_tool.pt:7]: Is this needed?

File: src/euphorie/client/browser/templates/oira_tool.pt [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      meta:interpolation="true"
```
Comment (ale-rt) [inline src/euphorie/client/browser/templates/oira_tool.pt:8]: Not needed

File: src/euphorie/client/browser/templates/oira_tool.pt [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+      metal:use-macro="context/@@shell/macros/shell"
+      tal:define="
+        webhelpers nocall:context/@@webhelpers;
```
Comment (ale-rt) [inline src/euphorie/client/browser/templates/oira_tool.pt:11]: Not needed

**Review by ale-rt — CHANGES_REQUESTED** [review]

**Comment by pysailor** [comment]
Thanks, I removed all the unnecessary stuff.

**Review by ale-rt — APPROVED** [review]

**Pull request merged**